omnibus/config/software/elasticsearch.rb

```diff
@@ -30,6 +30,11 @@
          sha512: "9f6179ee49baa48b49c5328b88ddf2f0ef868f49c1f04d77975622120749725c48ac09cd565c05f6033eb227eaff905aff9f881a85efcf2fa75cc586cb8c45cb"
 end
 
+version "7.9.1" do
```

Comment:
I think omnibus/config/software/elasticsearch.rb can be removed from chef-server, since it is present in omnibus-software.

Reply:
The entire file can be removed?  Or do you mean remove the commit where I made a change?

Reply:
yeah ref: https://github.com/chef/omnibus-software/pull/1280

As a note, the sha512 that I see is different from the one here:

```
➜  Downloads cat elasticsearch-7.9.1-linux-x86_64.tar.gz.sha512
e24aab0fbeb0b53cc386bb0ca1fc84c457851c5d80d147324bf97ff42f063332a93dec3c693550662393a72c7a0522a100181dd9a7d50b3e487a0f2a2a9bbcc0  elasticsearch-7.9.1-linux-x86_64.tar.gz%
```